packages/web-app-files/tests/unit/components/Search/List.spec.js

```diff
@@ -0,0 +1,189 @@
+      const noContentMessage = wrapper.find(selectors.noContentMessage)
+      const filesTable = wrapper.find(selectors.filesTable)
+
+      expect(filesTable.exists()).toBeFalsy()
+      expect(noContentMessage.exists()).toBeTruthy()
+      expect(noContentMessage.text()).toEqual('No search term entered')
+    })
+    it('should not show files table', () => {
+      const filesTable = wrapper.find(selectors.filesTable)
+
+      expect(filesTable.exists()).toBeFalsy()
+    })
+  })
+
+  describe('when resource is not found', () => {
+    let wrapper
+    beforeEach(() => {
+      wrapper = getWrapper('epsum')
+    })
+
+    it('should show text "No resource found"', () => {
+      const noContentMessage = wrapper.find(selectors.noContentMessage)
+      const filesTable = wrapper.find(selectors.filesTable)
+
+      expect(filesTable.exists()).toBeFalsy()
+      expect(noContentMessage.exists()).toBeTruthy()
+      expect(noContentMessage.text()).toEqual('No resource found')
+    })
+    it('should not show files table', () => {
+      const filesTable = wrapper.find(selectors.filesTable)
+
+      expect(filesTable.exists()).toBeFalsy()
+    })
+  })
+
+  describe('when resources are found', () => {
+    const spyTriggerDefaultAction = jest
+      .spyOn(List.mixins[0].methods, '$_fileActions_triggerDefaultAction')
+      .mockImplementation()
+    const spyRowMounted = jest.spyOn(List.methods, 'rowMounted')
+
+    let wrapper
+    beforeEach(() => {
+      wrapper = getWrapper('lorem', files)
+    })
+
+    it('should not show no-content-message component', () => {
+      const noContentMessage = wrapper.find(selectors.noContentMessage)
+
+      expect(noContentMessage.exists()).toBeFalsy()
+    })
+    it('should show files table', () => {
+      const filesTable = wrapper.find(selectors.filesTable)
+      const pagination = wrapper.find(selectors.pagination)
+
+      expect(filesTable.exists()).toBeTruthy()
+      expect(filesTable.props().resources).toMatchObject(files)
+      expect(pagination.exists()).toBeTruthy()
+    })
+    it('should send correct props on list-info component', () => {
+      const listInfo = wrapper.find(selectors.listInfo)
+
+      expect(listInfo.exists()).toBeTruthy()
+      expect(listInfo.props().files).toEqual(files.length)
+      expect(listInfo.props().folders).toEqual(0)
+      expect(listInfo.props().size).toEqual(getTotalSize(files))
+    })
+    it('should call "$_fileActions_triggerDefaultAction" function on "fileClick" event', async () => {
+      const filesTable = wrapper.find(selectors.filesTable)
+      await filesTable.vm.$emit('fileClick')
+
+      expect(spyTriggerDefaultAction).toHaveBeenCalledTimes(1)
+    })
+    it('should call "rowMounted" function on "rowMounted" event', async () => {
+      const filesTable = wrapper.find(selectors.filesTable)
+      await filesTable.vm.$emit('rowMounted')
+
+      expect(spyRowMounted).toHaveBeenCalled()
```

Comment:
```suggestion
      expect(spyRowMounted).toHaveBeenCalledTimes(1)
```
will be more informative